### ISSUE TITLE
Not technically a issue - Mi Notebook Pro Enhanced Edition Hackintosh Compatibility

### ISSUE THREAD

**Issue by DavidIlie** [daliansky/XiaoMi-Pro-Hackintosh#320]
Hello everyone,

I am interested in purchasing a Mi Notebook Pro Enhanced Edition because it has the new i7 10 gen. 

But I've been having difficulty searching if anybody has had success with it. 
If anyone knows or is willing to help, please don't hesitate to contact me at: david@davidilie.com

Thanks,

**Comment by eminfedar** [comment]
I'm curious about this too. Please fill this issue with information. Thanks!

**Comment by dawidmalina** [comment]
I tried to install hackintosh on my brand new Mi Notebook Pro Enhanced (i7-10510U) with latest release but booting stuck on apple logo. 

When i've booted with `-v -x -s -no-zp` process stopped with fallowing message:
```
ApplePS2Controller: Notification consumer published: ApplePS2Controller
ApplePS2Controller: Notification consumer published: ApplePS2Keyboard
```

Full screen info:
![20200327_154119](https://user-images.githubusercontent.com/5910851/77831500-d2aa5000-712f-11ea-87a6-ea2497fe5ab2.jpg)

**Comment by eminfedar** [comment]
@dawidmalina removing Voodoo kexts solving this issue, you should also apply acpi patch.

I have installed the system but haven't managed to work touchpad & brightness settings. Other things are working. If I manage to work it, I will share the fully working EFI, but it is hopeless & I don't have that much information about hackintosh :(

**Comment by dawidmalina** [comment]
@eminfedar I'm glad you were able to solve this issue 👍 

I assuming you fallowed with this: https://www.tonymacx86.com/threads/stuck-at-appleps2controller-elan-touchpad.282395/page-2 and https://www.tonymacx86.com/threads/asus-rog-strix-z390-e-gaming.282709/page-2#post-1998962

Did you removed all `Voodo*` kexts or just `VoodooPS2Controller.kext`? I'm asking as I've patched acpi and remove `VoodooPS2Controller.kext` but I've ended up with KP :(

**Comment by dawidmalina** [comment]
![20200328_225416](https://user-images.githubusercontent.com/5910851/77834780-676c7800-7147-11ea-8c9e-f187e6e75319.jpg)

**Comment by eminfedar** [comment]
Yes remove All Voodo and use only these args: `darkwake=0 gfxrst=1 -v`

Also I used with OpenCore.

**Comment by dawidmalina** [comment]
I was able to run installer but my hard drive is not recognised. Are you using original one or you have new one?

**Comment by eminfedar** [comment]
I hava macos from another website, not original. Also during installation you need a usb keyboard and mouse :)

Btw the macOS I used:https://osxinfo.net/konu/macos-mojave-10-14-6-amd-intel-kurulum-imaji.9570/

Do not use their EFI, but only the system.

**Comment by dawidmalina** [comment]
For the brightness issue there is some suggestion here: https://github.com/daliansky/XiaoMi-Pro-Hackintosh/issues/353 but i didn't try it yet.

**Comment by agassecond** [comment]
So, anyone got working config for this model?
Mine is semi woring with itwln wireless but without gestures still...

**Comment by Slash137** [comment]
> So, anyone got working config for this model?
> Mine is semi woring with itwln wireless but without gestures still...

My config works except for wifi, bt, native speaker, fprint, mx250...
Do you have a working Intel wireless driver for 2020 enhanced version?
Can you share please? ty

**Comment by sciro24** [comment]
> > So, anyone got working config for this model?
> > Mine is semi woring with itwln wireless but without gestures still...
> 
> My config works except for wifi, bt, native speaker, fprint, mx250...
> Do you have a working Intel wireless driver for 2020 enhanced version?
> Can you share please? ty

I have your situation, you have some news ? for me doesn't work ( audio , bt  and wifi )

**Comment by MiFrance** [comment]
> I have your situation, you have some news ? for me doesn't work ( audio , bt and wifi )

The same issues for me :(
